.appveyor.yml

```diff
@@ -16,7 +16,8 @@ install:
 # To run Nodejs workflow integ tests
 - ps: Install-Product node 8.10
 
-- "set PATH=%PYTHON%\\Scripts;%PYTHON%\\bin;%PATH%"
+- "set PATH=%PYTHON%;%PYTHON%\\Scripts;%PYTHON%\\bin;%PATH%"
+- "echo %PATH%"
```

Comment:
When you rebase: I would move this echo down the script so we see everything in the PATH.

Reply:
sounds good.